resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStepSuite.scala

```diff
@@ -249,6 +249,18 @@ class BasicExecutorFeatureStepSuite extends SparkFunSuite with BeforeAndAfter {
       executor.container, SecurityManager.ENV_AUTH_SECRET))
   }
 
+  test("Mount spark driver's conf dir as configmap on executor pod's container.") {
```

Comment:
? This looks like a little misleading. It's a general `conf` dir instead of `driver's conf dir`, isn't it?

Reply:
Driver's `spark.properties` file's content is effective of what user my specify programmatically(pass as cli options) and pass as a spark.conf file. Therefore we use the same config map, and mount on the executors as well. So technically, it is correct to say, we mount driver's conf dir on executor pod's as well.

Reply:
However, if you think it is misleading, can you tell how?

Reply:
This PR aims to `Support propagating SPARK_CONF_DIR files to driver and executor pods`, not copying from `driver conf dir` to `executor` conf dir. Although the content is the same, it's not the same one.

Reply:
How about, if I change it to `Verify spark conf dir is mounted as configmap volume on executor pod's container.`